README.md

```diff
@@ -1 +1,5 @@
-This is an uzhgorod2000@yahoo.com homework repository (file in "main")
\ No newline at end of file
+<<<<<<< HEAD
+This is an uzhgorod2000@yahoo.com homework repository (file in "main")
+=======
+This is an uzhgorod2000@yahoo.com homework repository (file in HW_02)
+>>>>>>> HW_02
```

Comment:
Here is the version conflict of several changes in one file. Change it and commit again